ansible/roles/hoover/tasks/main.yml

```diff
@@ -157,9 +157,13 @@
     dest: /opt/hoover/snoop/snoop/site/settings/testing_local.py
 
 - name: Create the local.py config for snoop
-  copy:
+  template:
     src: snoop-local.py
     dest: /opt/hoover/snoop/snoop/site/settings/local.py
+    owner: liquid-apps
```

Comment:
Does `liquid-apps` need write permissions on this file? If not, let's keep it root-owned.